### ISSUE TITLE
"Wait for all downloads" in Render to Video does not wait for all downloads

### ISSUE THREAD

**Issue by astrodavid10** [WorldWideTelescope/wwt-windows-client#201]
I am not certain if selecting "Wait for all downloads" when rendering a tour to video (really frames) actually works in any instance, but it definitely does not if a HiPS dataset is the one being rendered.

**Comment by astrodavid10** [comment]
I may have jumped the gun on what to call this... in looking at my render, the data tiles in at the appropriate zoom level and then momentarily backtracks and drops a chunk of the view to a much lower resolution. This is in a 3840x2160 screen size at 100% Windows scaling and at 60 fps

**Comment by pkgw** [comment]
I think I get what you're describing. Does the resolution glitch last, like a single frame, or longer than that?

My guess is that this one might be tricky to trigger so if you're able to report any more notes about how to reproduce the issue consistently, that would probably be very helpful.

**Comment by astrodavid10** [comment]
I don't think that this is only a HiPS problem, but it might be HiPS exacerbated. In this render, it lasts 40 frames which is 2/3 of a sec.
Pre glitch:
![orion_dss_unwise_v1____2919](https://user-images.githubusercontent.com/65675418/149859141-8d71bbdf-12b0-48f5-b054-138d3922bc84.jpg)
Glitch:
![orion_dss_unwise_v1____2920](https://user-images.githubusercontent.com/65675418/149859156-aad5c8b3-9b78-4cb3-b59e-46cb2cc724f2.jpg)
.

**Comment by astrodavid10** [comment]
I'll try a couple re-renders to see if it is reproducible and always in the same spot.

**Comment by pkgw** [comment]
Interesting! CC @imbasimba if you happen to have the chance to think about why this might be happening.
